lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -47,6 +48,17 @@ def json_error(file)
           ValidationError.new('translation.not_json', file: file.relative_path, errors: e)
         end
 
+        def required_keys(file)
+          return nil unless file.relative_path == 'translations/en.json'
```

Comment:
You don't need the `nil` here, can just be `return unless file.relative_path == 'translations/en.json'`.

Could even combine this with the guard below into something like:
`return if file.relative_path != 'translations/en.json' || JSON.parse(file.read)['app']['name']`